src/redis.go

```diff
@@ -59,7 +60,13 @@ func main() {
 		os.Exit(0)
 	}
 
-	conn, err := newRedisCon(args.Hostname, args.Port, args.UnixSocketPath, args.Password)
+	renamedCommands := make(map[string]string)
+	if args.RenamedCommands.Get() != nil {
+		renamedCommands = getRenamedCommands(args.RenamedCommands)
+		log.Debug("Parsed renamed redis commands: %#v", renamedCommands)
+	}
+
+	conn, err := newRedisCon(args.Hostname, args.Port, args.UnixSocketPath, args.Password, renamedCommands)
```

Comment:
As mentioned at the beginning, I think it's better to leave `renamedCommands` out of the constructor and just assign it later (for example, inside the if).

Reply:
agreed, I have moved it out and is now populating `renamedCommands map[string]string` afterwards when `args.RenamedCommands` is defined